src/chains/ethereum/ethereum/package.json

```diff
@@ -72,8 +72,8 @@
     "env-paths": "2.2.1",
     "eth-sig-util": "2.5.3",
     "ethereumjs-abi": "0.6.8",
-    "ethereumjs-util": "7.1.4",
-    "keccak": "3.0.1",
+    "ethereumjs-util": "7.1.5",
+    "keccak": "3.0.2",
```

Comment:
`@ethereumjs/*` packages have updated to this version of `keccak`, so I matched our dependency with theirs so we don't have to install multiple.